src/info.rs

```diff
@@ -262,6 +279,27 @@ impl Info {
         authors
     }
 
+    fn get_git_info(dir: &str) -> (String, String){
+        let version = Command::new("git")
+            .arg("--version")
+            .output()
+            .expect("Failed to execute git.");
+        let version = String::from_utf8_lossy(&version.stdout);
+        let version = version.to_string().replace('\n',"");
+
+        let username = Command::new("git")
+            .arg("-C")
+            .arg(dir)
+            .arg("config")
+            .arg("--get")
+            .arg("user.name")
+            .output()
+            .expect("Failed to execute git.");
+        let username = String::from_utf8_lossy(&username.stdout);
+        let username = username.to_string().replace('\n',"");
```

Comment:
Even though the return type of `from_utf8_lossy` is `Cow<str>`, you can treat it as if it was a `&str`, so you don't need `to_string`.
See [this example](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&code=fn%20main()%20%7B%0A%20%20%20%20println!(%0A%20%20%20%20%20%20%20%20%22Using%20%60replace%60%20without%20%60to_string%60!%20%7B%3A%3F%7D%22%2C%0A%20%20%20%20%20%20%20%20String%3A%3Afrom_utf8_lossy(%26%5B100%2C%20121%5D).replace(%27d%27%2C%20%22x%22)%2C%0A%20%20%20%20)%3B%0A%7D).

Reply:
Thank you for that example, I've fixed both places that I did that.